### ISSUE TITLE
`--cert-selector` with string argument incorrectly fails when a Value includes a space

### ISSUE THREAD

**Issue by chalford** [aws/rolesanywhere-credential-helper#82]
Apologies if I have missed some documentation that describes how to do this.

`getStringMap()` in `credentials.go` does `strings.Split(s, " ")` on the `--cert-selector` argument. If the `Value` is a DN that includes components with spaces, these get split incorrectly.

Perhaps either of these would be a way to approach the problem:
1. The --cert-selector argument provides some way to substitute spaces with another character, or,
2. The --cert-selector argument uses a different separator character (than space) for each key-value pair

A workaround exists by using the `file://` argument with a JSON file, where a space in the `Value` field works as expected.

**Comment by 13ajay** [comment]
You're right, the current workaround is to use a file as the selector as you said. I had started to address this issue in #66, but that requires some updates at this point. I'll discuss further with my colleagues and hopefully address it soon.
